src/main/scala/com/actian/spark_vector/datastream/DataStreamClient.scala

```diff
@@ -47,17 +51,36 @@ case class DataStreamClient(vectorProps: VectorConnectionProperties, table: Stri
   private def prepareUnloadSql(table: String) = s"prepare for x100 stream from $table"
   private def startUnloadSql(selectQuery: String) = s"insert into external table $selectQuery"
 
-  private def executeSql(sql: String): Future[Int] = Future { jdbc.executeStatement(sql) }
+  /** Execute a sql (statement) within a future task */
+  private def executeSql(sql: String): Future[Int] = Future { jdbc.executeStatement(sql) } match {
+    case f =>
+      f onFailure {
+        case t =>
+          logError(t.stackTrace)
```

Comment:
I think it's more important to print the error message than the stack trace(the stack trace will just show it was called from here and thrown in some jdbc place). Also, doesn't logError("Some error message", t) print the stack trace automatically?